counter-sink/README.adoc

```diff
@@ -0,0 +1,36 @@
+= Spring Cloud Stream Counter Sink
+
+A *Spring Cloud Stream* module that simply counts the number of messages it receives,
+optionally storing counts in a separate backend such as redis.
+== Requirements
+
+To run this module, you will need to have installed:
+
+* Java 7 or Above
+
+This example requires Redis to be running on localhost.
```

Comment:
avoid the word "example" - this is a module

also, notice the simplified bullet point approach I used for time-source and log-sink (was inspired to do that for what should end up in an archetype, etc)... for example: https://github.com/spring-cloud/spring-cloud-stream-modules/tree/master/time-source